governance/reviews/template.md

```diff
@@ -111,17 +200,17 @@ Governance Drift can occur when the executed and observable governance of a proj
 
 The project [does/does] not experience governance drift as indicated by...
 
-#### Ownership
+### Ownership
```

Comment:
no, this was on purpose.  Ownership is linked to Drift, rather than being separate from it

Reply:
I think you mean Ownership is linked to "Operation", but I got it.